src/main/scala/no/ndla/conceptapi/service/ReadService.scala

```diff
@@ -30,13 +36,55 @@ trait ReadService {
       }
 
     def publishedConceptWithId(id: Long, language: String, fallback: Boolean): Try[api.Concept] =
-      publishedConceptRepository.withId(id) match {
+      publishedConceptRepository.withId(id).map(addUrlOnVisualElement) match {
```

Comment:
Jeg syns vi skal legge på `addUrlOnVisualElement` for `conceptWithId` også slik at de oppfører seg likt :slightly_smiling_face: